src/jesse_lib.erl

```diff
@@ -35,11 +35,36 @@
         , get_schema_id_key/1
         , get_schema_id/1
         , get_schema_id/2
+        , json_encode/1
+        , json_decode/1
         ]).
 
 %% Includes
 -include("jesse_schema_validator.hrl").
 
+%% Use new json library if available
+-ifdef(OTP_RELEASE).
+  -if(?OTP_RELEASE >= 27).
+  %% OTP 27 or higher
+json_decode(Bin) ->
+    json:decode(Bin).
+json_encode(Bin) ->
+    json:encode(Bin).
+  -else.
+  %% OTP 26 to 21.
+json_decode(Bin) ->
+    jsx:decode(Bin).
```

Comment:
I'm only afraid this will have suboptimal performance, but well, I guess that if performance is critical it can run an upgrade to OTP27, jsx was lagging behind long ago anyway :)